grakn-graql/src/main/java/ai/grakn/graql/internal/query/ConceptBuilder.java

```diff
@@ -90,14 +90,38 @@
     private final Map<BuilderParam<?>, Object> params = new HashMap<>();
```

Comment:
can we compromise to `preProvidedParams` then? At least then when there is a mismatch between the `usedParams` and `preProvidedParams` resulting in a throw it's a bit more clear why ? 

`params` says nothing. It's kind of one of the most generic names possible. Kind of like `fileds` :P